plugins/html-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/reflection/HtmlReflection.java

```diff
@@ -320,21 +248,83 @@ public static HtmlElement findBlock(Page page, String blockPath) throws NoSuchEl
     }
 
     /**
-     * See {@link HtmlReflectionUtils#findBlocks(String, Object, boolean)} for description.
+     * See {@link HtmlReflection#findBlocks(String, Object, boolean)} for description.
      * Search is being performed on a current page
      *
      * @param page the page on which the method will be executed
      * @param blockPath full path or just a name of the block to search
      * @return list of objects that were found by specified path
      */
-    public static List<HtmlElement> findBlocks(Page page, String blockPath) throws NoSuchElementException {
+    public List<HtmlElement> findBlocks(Page page, String blockPath) throws NoSuchElementException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the declaration of thrown exception 'org.openqa.selenium.NoSuchElementException' which is a runtime exception. ([squid:RedundantThrowsDeclarationCheck](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3ARedundantThrowsDeclarationCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=101&issues=AWbDgFYRLpqqSUlctfyD&open=AWbDgFYRLpqqSUlctfyD)

[ISSUE_KEY]: AWbDgFYRLpqqSUlctfyD